packages/beacon-push-draw-id/src/handler.ts

```diff
@@ -0,0 +1,35 @@
+import { Relayer } from 'defender-relay-client';
+import { DefenderRelayProvider, DefenderRelaySigner } from 'defender-relay-client/lib/ethers';
+import { getContracts, beaconPushDrawId } from '@pooltogether/v4-autotasks-library';
+import { mainnet, testnet } from '@pooltogether/v4-pool-data';
+
+export async function handler(event: any) {
+  const provider = new DefenderRelayProvider(event);
+  const signer = new DefenderRelaySigner(event, provider, { speed: 'fast' });
+  const relayer = new Relayer(event);
+
+  const chainId = Number(process.env.CHAIN_ID);
+  const contracts = getContracts(chainId, mainnet, testnet);
+
+  try {
+    const transactionPopulated = await beaconPushDrawId(contracts, {
```

Comment:
What we're really doing here is pushing the prize distribution.
So I would rename the package and helpers function to: `beacon-push-prize-distribution`
We should also rename `draw-lock-and-push` to `beacon-lock-and-push`. It would make more sense since we also have `receiver-lock-and-push`.